lib/recurly/apple-pay.js

```diff
@@ -238,7 +262,17 @@ class ApplePay extends Emitter {
   onPaymentAuthorized (event) {
     debug('Payment authorization received', event);
 
-    this.recurly.request('post', '/apple_pay/token', event.payment.token, (err, token) => {
+    // TODO: can we be sure an apple pay form will always have the number field?
+    // var selector = this.recurlyconfig.fields.number.selector;
+    var selector = '[data-recurly=first_name]'
+    var form = dom.findNodeInParents(window.document.querySelector(selector), 'form');
+    var data = normalize(fields, form);
+    var inputs = data.values || {};
```

Comment:
I think we should have the form element passed in to the ApplePay class on init in order to avoid this loose coupling of the `data-recurly` convention. `options.form` seems best.